generators/liquibase/templates/gradle/liquibase.gradle.ejs

```diff
@@ -43,28 +44,39 @@ if (!project.hasProperty("runList")) {
  project.ext.runList = "main"
 }
 
+<%_ 
+const defaultSchemaName = devDatabaseTypeMysql && baseName ? baseName : ""
```

Comment:
Please move to
https://github.com/jhipster/generator-jhipster/blob/eeea6f2371d6dd7d978f4956cf82b51db6a10eda/generators/liquibase/generator.mts#L79

like:
```js
preparing({ application }) {
  application.liquibaseDefaultSchemaName = application.databaseTypeSql && application.devDatabaseTypeMysql ? application.baseName ? application.baseName : '';
},
```

We need to pass to mavenPlugin too:
https://github.com/jhipster/generator-jhipster/blob/eeea6f2371d6dd7d978f4956cf82b51db6a10eda/generators/liquibase/generator.mts#L253